src/prelude/mod.rs

```diff
@@ -89,16 +89,20 @@ macro_rules! make_block(
 #[macro_export]
 macro_rules! make_fp_block(
     ($size:ident) => {
-        if cfg!(feature = "uniform-random") {
```

Comment:
Can you elaborate on what/why the diff in this file does? It looks like the logic is unchanged, with boolean cfg switched to attribute cfg. https://doc.rust-lang.org/rust-by-example/attribute/cfg.html suggests there is no difference between those two.

Reply:
`cfg!` will only be expand into a bool. when `uniform-random` feature is turned off, the compilation will fail because `rand` cannot be found.